dali/dafilesrv/dafilesrv.cpp

```diff
@@ -332,6 +333,10 @@ int initDaemon()
 int main(int argc,char **argv) 
 {
     InitModuleObjects();
+    IDaFileSrvHook *remoteHook = queryDaFileSrvHook();
+    if (remoteHook)
+        removeIFileCreateHook(remoteHook);
+
```

Comment:
Trivial: a comment about removing all hooks at this lowest point might be helpful.

Reply:
Will add a comment.